src/tools/convert_dgv.py

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env python
+import argparse
+import logging
+import itertools
+from typing import Dict, Tuple, List
+from mavis.constants import COLUMNS, ORIENT, STRAND, SVTYPE
+import pandas as pd
+from mavis.error import InvalidRearrangement
+
+from mavis.annotate.file_io import parse_annotations_json
+from mavis.annotate.base import ReferenceName
+from mavis.breakpoint import Breakpoint, BreakpointPair
+from mavis.convert import TRACKING_COLUMN
+from mavis.util import output_tabbed_file
+
+"""
```

Comment:
This should be at the top of the file

Reply:
Moved in [d640ca7](https://github.com/bcgsc/mavis/pull/336/commits/d640ca7d5c787d290086f19c0779743277a26b01)